.agents/skills/code-simplifier/SKILL.md

```
@@ -0,0 +1,87 @@
+```bash
+git diff --name-only HEAD~1  # Files changed in last commit
+git diff --name-only         # Uncommitted changes
+```
+
+If the user specifies particular files or directories, use those instead.
+
+### Step 2: Launch the Subagent
+
+Invoke the `code-simplifier` subagent via the Task tool with `subagent_type: "code-simplifier"`:
+
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)